config/500-webhook.yaml

```diff
@@ -0,0 +1,62 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: webhook
+  namespace: cloud-run-events
+  labels:
+    events.cloud.run/release: devel
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: webhook
+      role: webhook
+  template:
+    metadata:
+      labels:
+        app: webhook
+        role: webhook
+        events.cloud.run/release: devel
+    spec:
+      serviceAccountName: controller
+      containers:
+        - name: webhook
+          # This is the Go import path for the binary that is containerized
+          # and substituted here.
+          image: github.com/GoogleCloudPlatform/cloud-run-events/cmd/webhook
+          imagePullPolicy: Always
+          resources:
+            requests:
+              cpu: 20m
+              memory: 20Mi
+            limits:
```

Comment:
In other files, unless we know for sure what the limits are, we should not just wag it. If these are known to be good limits, then it's fine to leave them in.